assets/scss/_sidebar.scss

```diff
@@ -52,6 +52,8 @@
   }
 
   @media only screen and (min-width: 420px) {
+    height: 2px;
+
```

Comment:
Before and after
<img width="145" alt="Screenshot 2024-10-09 at 7 13 06 PM" src="https://github.com/user-attachments/assets/c1ac0541-f0fd-4643-bddd-05230841dd0b">
<img width="133" alt="Screenshot 2024-10-09 at 7 13 46 PM" src="https://github.com/user-attachments/assets/ec87354a-7fa3-4f03-9c9e-1dd38994b308">